src/main/kotlin/app/manager/service/ManagerService.kt

```diff
@@ -91,39 +91,48 @@ class ManagerService
 
             val bucketResponseEntity = assetStoreApi.getSnippet(snippetKey)
 
+            // Missing getting the author from the permissions service
```

Comment:
Are we waiting for the permissions endpoint for this?

Reply:
sip, lo hice antes de hacer lo del permissions entonces me di cuenta que tenía que poner el llamado ahi

Reply:
Flaca rompete un ingles

Reply:
Cuando este te lo apruebo

Reply:
> Flaca rompete un ingles
Estoy modo Copa América yo hablo español 🇦🇷